src/components/Scaffolding/Scaffolding.scss

```diff
@@ -0,0 +1,88 @@
+@import 'vars';
+@import 'mixins';
+@import 'functions';
+
+/* fiord, elephant, alto, malibu, concrete, mine-shaft */
+
+
+.ais-Hits-list  {
+    list-style: none;
+    display:flex;
+    flex-direction:column;
+    justify-content: flex-end;
+}
+.ais-SearchBox-input {
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+    padding: .3rem 1.7rem;
```

Comment:
Can we use `em` as the rest of the modules?

Reply:
yes, fixed